apps/smtp/src/modules/event-handlers/use-case/send-event-messages.use-case.ts

```diff
@@ -68,7 +68,7 @@ export class SendEventMessagesUseCase {
     const eventSettings = config.events.find((e) => e.eventType === event);
 
     if (!eventSettings) {
-      this.logger.info("Configuration found but settings for this event are missing");
+      this.logger.info("Configuration found but settings for this event are missing", { event });
```

Comment:
**Suggestion:** you can use `loggerContext` to set event for all logs from this use case:

```tsx
loggerContext.set(`eventName`, event)
```
This `eventName` can be extracted into constans so it will be consistent with other places 😄